Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -326,6 +335,74 @@ open class MapView: UIView {
             mapboxMap: mapboxMap)
     }
 
+    private func subscribeToLifecycleNotifications() {
+        if bundle.infoDictionary?["UIApplicationSceneManifest"] != nil {
+            if #available(iOS 13.0, *) {
```

Comment:
You can combine these into a single if statement and avoid the duplication in the else blocks.

Reply:
TIL 😲